### PR TITLE
new feature block object-storage permissions command

### PR DESCRIPTION
**Pull request by caberos**
[#issue1667](https://github.com/softlayer/softlayer-python/issues/1667)

image

### CODE REVIEW

File: SoftLayer/CLI/block/object_storage_permission.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('object_id')
+@environment.pass_env
+def cli(env, object_id):
+    """Display details for a cloud object storage."""
```
Comment (BrianSantivanez) [inline SoftLayer/CLI/block/object_storage_permission.py:14]: Update command description

File: SoftLayer/CLI/block/object_storage_permission.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                                  'type',
+                                  'url'])
+    for end_point in end_points:
+        table_url.add_row([end_point.get('region') or '',
+                           end_point.get('location') or '',
+                           end_point.get('type'),
+                           end_point.get('url'), ])
+
+    table.add_row(['UUID', cloud.get('uuid')])
+    table.add_row(['Credemtial', table_credentials])
```
Comment (BrianSantivanez) [inline SoftLayer/CLI/block/object_storage_permission.py:42]: Replace `Credemtial` by `Credential`, and maybe like object storage could have many credentials, it is better to use the plural form

File: SoftLayer/CLI/block/object_storage_permission.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    table.add_row(['EndPoint URL', table_url])
```
Comment (BrianSantivanez) [inline SoftLayer/CLI/block/object_storage_permission.py:43]: Maybe like object storage could have many endpoint URL´s, it is better to use `EndPoint URL´s`

**Comment by edsonarios** [comment]
Screenshots of the test result
image
image

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**